dags/ltv.py

```diff
@@ -101,3 +101,17 @@
         dag=dag,
     )
     wait_for_search_clients_last_seen >> ltv_daily
+
+response = urlopen('/'.join([
+    'https://raw.githubusercontent.com/mozilla/bigquery-etl/master/sql',
+    'revenue_derived', 'client_ltv_v1', 'query.sql']))
```

Comment:
I think this will break the dag if it doesn't url doesn't exist (like it does now).

Reply:
The import is also missing.